configurator/src/main/resources/topTabScene.fxml

```diff
@@ -19,29 +19,29 @@
        <FlowPane alignment="TOP_LEFT" style="-fx-background-color: #ffffff" >
            <VBox>
                <padding>
-                   <Insets top="0" bottom="0" left="0" right="20"></Insets>
+                   <Insets top="0" bottom="0" left="20" right="20"></Insets>
                </padding>
                <ImageView fx:id="picmgLogo" pickOnBounds="true" fitHeight="100" fitWidth="100" preserveRatio="true" >
                    <image>
                        <Image url="@picmg_logo.png" backgroundLoading="true"/>
                    </image>
                </ImageView>
            </VBox>
+           <Label prefHeight="100" text = "PICMG Smart Sensor Configurator" textFill="black" style="-fx-font-weight: bold;">
```

Comment:
I believe Mr. Sandy asked that we drop "PICMG" from the title, asthe logo is now directly antecedent.